models/book.js

```diff
@@ -0,0 +1,37 @@
+import mongoose from 'mongoose'
+
+const Schema = mongoose.Schema
+
+const BookSchema = new Schema({
+  bookID: {
+    type: Number
+  },
```

Comment:
Mongoose adds unique _id field so it is no need to add one by hand. This will make your work harder when you will start adding new records to the database later.

Reply:
Removed it :)